testutils/utils.go

```diff
@@ -59,6 +61,22 @@ func WipeEtcd() {
 		panic(err)
 	}
 	_ = be.Clean()
+
+	// Set the ready flag so calls to the CNI plugin can proceed
+	calicoClient, _ := client.NewFromEnv()
+	ctx := context.Background()
+	newClusterInfo := api.NewClusterInformation()
+	newClusterInfo.Name = "default"
+	newClusterInfo.Spec.CalicoVersion = "v0.0.0"
+	newClusterInfo.Spec.ClusterType = "test"
+	newClusterInfo.Spec.ClusterGUID = fmt.Sprintf("%s", hex.EncodeToString(uuid.NewV4().Bytes()))
+	datastoreReady := true
+	newClusterInfo.Spec.DatastoreReady = &datastoreReady
+	ci, err := calicoClient.ClusterInformation().Create(ctx, newClusterInfo, options.SetOptions{})
```

Comment:
you can just put `context.Background()` here since the `ctx` variable is not used anywhere else